onboard/nav/rover.cpp

```diff
@@ -265,6 +273,15 @@ const double Rover::longMeterInMinutes() const
     return mLongMeterInMinutes;
 }
 
+void Rover::updateRepeater(RadioSignalStrength& radioSignal)
+{
+    (radioSignal.signal_strength <= mRoverConfig[ "radioRepeater" ][ "lowSignal" ].GetDouble()) ? missedPings += 1 : missedPings = 0;
+}
+
+bool Rover::isDropRepeater()
+{
+    return (missedPings >= mRoverConfig[ "radioRepeater" ][ "threshold" ].GetDouble());
```

Comment:
To clarify. If we get a low signal, are you picturing that the rover stops moving for a certain amount of time to see if the signal is consistently low? or is it an internal clock and the rover can continue moving? Part of the reason I chose pings was because the rover could keep moving, but maybe that's not a good idea.